src/System.Diagnostics.Process/tests/Performance/Perf.Process.cs

```diff
@@ -109,15 +180,23 @@ public void GetStandardOutput()
             StreamReader result;
             foreach (var iteration in Benchmark.Iterations)
             {
-                using (Process proc = CreateProcess())
+                // Create several processes to test on
+                Process[] processes = new Process[500];
+                for (int i = 0; i < 500; i++)
                 {
-                    proc.StartInfo.RedirectStandardOutput = true;
-                    proc.Start();
-                    
-                    using (iteration.StartMeasurement())
-                        result = proc.StandardOutput;
-                    proc.Kill();
+                    processes[i] = CreateProcess();
+                    processes[i].StartInfo.RedirectStandardOutput = true;
+                    processes[i].Start();
                 }
+
+                // Begin Testing
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < 500; i++)
+                        result = processes[i].StandardOutput;
```

Comment:
Update pushed; mind taking a look at the GetStandardOutput method? It essentially measures ReadToEnd() now, but it seems like it might be dependent on the execution time of the process and not be testing the correct thing.